extensions/sql-database-projects/src/dialogs/deployDatabaseQuickpick.ts

```diff
@@ -70,6 +70,33 @@ export async function launchDeployAppIntegrationQuickpick(project: Project): Pro
 	};
 }
 
+async function launchEulaQuickPick(baseImage: string): Promise<string | undefined> {
+	const baseImages = uiUtils.getDockerBaseImages();
+	const imageInfo = baseImages.find(x => x.name === baseImage);
+	let result: string | undefined = undefined;
+	if (imageInfo?.agreementInfo?.link) {
+		const link = imageInfo?.agreementInfo?.link;
+
+		// Ask user to update app settings or not
+		//
+		let choices = [
+			constants.yesString,
+			constants.noString
+		];
+		let options: vscode.QuickPickOptions = {
+			placeHolder: `${uiUtils.getAgreementDisplayText(imageInfo?.agreementInfo)} (${link.url})`,
```

Comment:
Having the URL as a placeholder is pretty useless to the user. I would suggest adding an action to the quickpick to open the URL when clicked